.github/workflows/release.yaml

```
@@ -40,7 +41,7 @@ jobs:
         env:
           TWINE_USERNAME: __token__
           TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
-        run: twine upload --verbose dist/* || echo 'Version exists'
+        run: twine upload --verbose dist/* || echo 'Version probably exists'
```

Lets actually get a failed workflow if this fails for any other reason than the version exist.
```suggestion
        run: twine upload --skip-existing --verbose dist/*
```

---

Oooooh like it 🤩